addons/website_blog/views/website_blog_templates.xml

```diff
@@ -94,39 +97,35 @@ list of filtered posts (by date or tag).
     <xpath expr="//div[@id='o_wblog_blog_top_droppable']" position="replace">
         <div class="container">
             <div class="row py-4">
-                <t t-if="blog">
-                    <div t-attf-class="mb-3 mb-md-0 #{'col-md-5' if (not opt_blog_list_view and not opt_blog_sidebar_show) else 'col-md-6'}">
-                        <t t-call="website.record_cover">
-                            <t t-set="_record" t-value="blog"/>
-                            <t t-set="additionnal_classes" t-value="'h-100 py-5 py-md-0 overflow-hidden rounded shadow'"/>
-                        </t>
-                    </div>
-                    <div t-att-class="'col-md-7' if (not opt_blog_list_view and not opt_blog_sidebar_show) else 'col-md-6'">
-                        <div class="container position-relative h-100 d-flex flex-column justify-content-around pt-1 pb-2">
-                            <div t-attf-class="o_wblog_post_title #{'js_tweet' if opt_blog_post_select_to_tweet else ''} #{'js_comment' if opt_blog_post_select_to_comment else ''}">
-                                <span t-field="blog.name" class="h1 d-block" placeholder="Blog's Title"/>
-                                <div t-field="blog.subtitle" class="h4" placeholder="Subtitle"/>
-                            </div>
+                <t t-set="_has_cover" t-value="blog._get_background(height=_resize_height, width=_resize_width) == 'none'"/>
+                <div t-att-class="'mb-3 mb-md-0 ' + ('css_non_editable_mode_hidden ' if _has_cover else '') + ('col-md-5' if (not opt_blog_list_view and not opt_blog_sidebar_show) else 'col-md-6')">
```

Comment:
Keep using `t-attf`, and you don't need to care about the combining spaces: `#{} #{} #{}`